cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,242 @@
+def get_salary_component(data,component,custom_field):
+    try:
+        data = json.loads(data)  
+
+        
+        salary_component = data.get("salary_component")
+        component_type=data.get("component_type")
+     
+
+
+        if int(component) == 1 and int(custom_field) == 1:
+            if salary_component:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)